sdk/eidos_sdk/system/processes.py

```diff
@@ -55,4 +56,39 @@ class ProcessDoc(MongoDoc):
     metadata: dict = {}
     agent: str
     state: str
-    data: Any
+    error_info: Optional[Any] = None
+
+
+async def store_events(agent: str, process_id: str, events: list[StreamEvent]):
+    try:
+        stored_events = []
+        for event_num, event in enumerate(events):
+            event_obj: Dict[str, Any] = {
+                **event.model_dump(),
+                "process_id": process_id,
+                "agent": agent,
+                "create_time": datetime.now().timestamp(),
```

Comment:
can we use isoformat here? it is human readable and lexicographically ordered

Reply:
I didn't intend this to be human readable as it should never get to the human side.